tensorflow_probability/python/distributions/laplace.py

```diff
@@ -202,6 +202,11 @@ def _median(self):
   def _mode(self):
     return self._mean()
 
+  def _quantile(self, x):
+    loc = tf.convert_to_tensor(self.loc)
+    scale = tf.convert_to_tensor(self.scale)
+    return tf.where(x > 0.5,loc - scale*tf.math.log(2*(1-x)), loc + scale*tf.math.log(2*x))
```

Comment:
You might need the double where trick similar to https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/internal/special_math.py#L151-L154 - would it be possible to write the laplace quantile function as some scaling of the Normal quantile function? that would make it a bit easier to implement and test I think.

Reply:
Not sure if I understand where the double where would be used.  As for writing the laplacian quantile function as a scaling of the Normal quantile function, a scaling *might* exist but I'm not immediately familiar with it.  I can look around for it.